src/components/PigeanTable.vue

```diff
@@ -33,11 +33,13 @@ export default Vue.component("pigean-table", {
         },
         sortBy() {
             return this.pigeanData.length === 0
-                ? 0
-                : this.config.sortBy
-                ? this.config.sortBy
-                : this.pigeanData[0]["combined"] !== undefined
-                ? "combined"
+                ? 0 
+                : this.config.fields.map(field => field.key).includes("factor_value") 
+                ? "factor_value" 
+                : this.config.sortBy 
+                ? this.config.sortBy 
+                : this.pigeanData[0]["combined"] !== undefined 
+                ? "combined" 
                 : "beta_uncorrected";
```

Comment:
If this gets any longer, consider refactoring this out as separate function. Otherwise, this will be very hard to read/understand.

Reply:
I think that makes sense.